chore: update dashboard to match new design

deboer-tim opened the pull request:
### What does this PR do?

The bootc extension dashboard should be updated to match the UX of the Kubernetes dashboard, but there is no reusable component yet [1]. We want to do a release soon, so there isn't time to wait for a component.

In order to avoid completely copy/paste/editing the current Kubernetes dashboard, I've added several temporary components to the /upstream folder:
- An expandable component that I've just opened a PR for [2], minus animation to reduce temporary dependencies.
- Resource and guide cards that are almost 100% copy/pasted from Podman Desktop.
- A DashboardPage that does a rough componentization of the Kubernetes dashboard. The final one might not look exactly like this, but it avoids most future-work within bootc.

And finally - an updated Bootc Dashboard that uses the component. Notes:
- The Bootc Images card has no link (since we plan to add a page next milestone)
- Disk image card links to the corresponding page.

[1] Dashboard component: https://github.com/podman-desktop/podman-desktop/issues/11363
[2] Expandable: https://github.com/podman-desktop/podman-desktop/pull/11364

### Screenshot / video of UI

<img width="800" alt="Screenshot 2025-02-28 at 8 59 14 AM" src="https://github.com/user-attachments/assets/605ed886-1086-4976-b804-e7fefa643f67" />

### What issues does this PR fix or reference?

Fixes #976.

### How to test this PR?

Open the dashboard!

ekidneyrh commented:
I agree with @cdrage that the icons on the cards are large. Maybe we could replicate the way they are displayed on this mockup:

https://github.com/podman-desktop/podman-desktop/issues/9704#issuecomment-2637408606

Icons are smaller and aligned to the left

deboer-tim commented:
Rebased to pick up svelte-check fix (png support).

Reduced guide card images by ~30%, going any further made the middle one look squished/tiny since it is square. The titles came directly from each site, but I've switched to sentence capitalization for consistency. Screenshot in the description updated, @cdrage let me know what you think.

deboer-tim commented:
I had missed Emma's comment earlier so just applied it now: smaller/logo images on guide card with left alignment, except for button. Image in the description updated.

cdrage reviewed (approved):
The files from upstream look 1-1, so fine that it doesn't have tests.

Went through the new Dashboard.svelte code, tested it as well as ran the tests locally too.

LGTM!

The page looks great and thanks for applying the recommended changes.

I also tested "building" and the "pulling" of the default image, and LGTM.

![Screenshot 2025-03-03 at 9 11 40 AM](https://github.com/user-attachments/assets/cb690288-46df-4d6c-baa4-e7c79ee8787d)

SoniaSandler commented on packages/frontend/src/lib/upstream/Expandable.svelte:
```diff
@@ -0,0 +1,48 @@
+    </div>
+  {/if}
+  {/if}
+  </div>
```
nit: EOF/ newline?

deboer-tim replied:
Doh. This file will be deleted as soon as we move up to new podman-desktop/ui-svelte library though, so no big deal.

SoniaSandler reviewed (approved):
LGTM, tested on Linux and it works. Amazing improvement from the old dashboard page!
![Screenshot from 2025-03-03 10-28-10](https://github.com/user-attachments/assets/79fe66a5-9c62-4618-916c-36c21a25421f)
